### PR TITLE
fzfnpm 0.6.0 (new formula)

### PR DESCRIPTION
**Pull request by FrancisVega**
Simple cli (bash) to run npm scripts available inside a package.json of the current directory if its available through fzf. This way you avoid looking inside the package.json or having to remember the name of the scripts.

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/fzfnpm.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+class Fzfnpm < Formula
+  desc "Fuzzy finder npm script runner"
+  homepage "https://github.com/FrancisVega/fzfnpm"
+  url "https://github.com/FrancisVega/fzfnpm/archive/refs/tags/v0.6.0.tar.gz"
+  sha256 "883cf5df0a76abbd22726609d8f516134e23350bfcb7fee5f2d29b1a5a0dc147"
+
```
Comment (chenrui333) [inline Formula/fzfnpm.rb:6]:
```suggestion
```

File: Formula/fzfnpm.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  license "MIT"
+
+  depends_on "bash"
```
Comment (chenrui333) [inline Formula/fzfnpm.rb:9]: can we use default installed bash or it has to be the latest bash?

File: Formula/fzfnpm.rb [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  depends_on "fzf"
+
+  def install
+    bin.install "bin/fzfnpm"
+  end
+
+  test do
+    assert_predicate Pathname.new("#{bin}/fzfnpm"), :executable?
```
Comment (chenrui333) [inline Formula/fzfnpm.rb:17]: Can you propose a more substantial test?

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula

File: Formula/fzfnpm.rb (hunk 2 above)
Comment (FrancisVega) [reply Formula/fzfnpm.rb:9]: We can use the default bash

File: Formula/fzfnpm.rb (hunk 3 above)
Comment (FrancisVega) [reply Formula/fzfnpm.rb:16]: Yes, I will do my best to do it thanks!

Comment (FrancisVega) [reply Formula/fzfnpm.rb:16]: I add a test to check if a package.json file exists. If not , the script exit with a message and exit code 1

**Comment by dawidd6** [comment]
```
==> brew audit fzfnpm --online --new-formula
==> FAILED
Full audit fzfnpm --online --new-formula output
  fzfnpm:
    * GitHub repository not notable enough (<[30](https://github.com/Homebrew/homebrew-core/actions/runs/3979445760/jobs/6821910447#step:6:31) forks, <30 watchers and <75 stars)
```

**Comment by SMillerDev** [comment]
Hi, this seems like a nice software, but not notable enough for us to package in Homebrew for now. This is not a decision on the merits of the software, but we cannot distribute every single piece of software and have to draw a line on “notability” somewhere. Of course, we would be happy to reconsider in the future.

Please consider hosting it in a personal tap, which is very easy to do: https://docs.brew.sh/How-to-Create-and-Maintain-a-Tap

**Pull request closed without merging**